docs/common/yum.md

```diff
@@ -1,167 +1,108 @@
-
-YUM Repositories
-================
+**YUM Repositories**
+====================
 
 About This Document
--------------------
+===================
 
 This document introduces YUM repositories and how OSG uses them.
 
 Repositories
-------------
+============
 
-OSG hosts four public-facing repositories at
-[repo.grid.iu.edu](http://repo.grid.iu.edu/):
+OSG hosts four public-facing repositories at [repo.grid.iu.edu](http://repo.grid.iu.edu/):
 
--   **release**: This repository contains software that we are willing
-    to support and can be used by the general community.
+-   **release**: This repository contains software that we are willing to support and can be used by the general community.
 -   **contrib**: RPMs contributed from outside the OSG.
--   **testing**: This repository contains software ready for testing. If
-    you install packages from here, they may be buggy, but we will
-    provide limited assistance in providing a migration path to a fixed
-    version.
--   **development**: This repository is the bleeding edge. Installing
-    from this repository may cause the host to stop functioning, and we
-    will not assist in undoing any damage.
-
-OSG’s RPM packages rely also on external packages provided by supported
-OSes and EPEL. You must have the following repositories available and
-enabled:
-
--   your OS repositories (SL 5/6/7, CentOS 5/6/7, or RHEL 5/6/7
-    repositories)
+-   **testing**: This repository contains software ready for testing. If you install packages from here, they may be buggy, but we will provide limited assistance in providing a migration path to a fixed version.
+-   **development**: This repository is the bleeding edge. Installing from this repository may cause the host to stop functioning, and we will not assist in undoing any damage.
+
+OSG's RPM packages rely also on external packages provided by supported OSes and EPEL. You must have the following repositories available and enabled:
+
+-   your OS repositories (SL 6/7, CentOS 6/7, or RHEL 6/7 repositories)
 -   EPEL repositories
--   the OSG repositories you’d like to use
+-   the OSG repositories you'd like to use
 
-If one of these repositories is missing you may have missing
-dependencies.
+If one of these repositories is missing you may have missing dependencies.
 
-<span class="twiki-macro WARNING"></span> We did not test other
-repositories. If you use packages from other repositories, like
-`jpackage`, `dag`, or `rpmforge`, you may encounter problems.
+!!! warning
+    We did not test other repositories. If you use packages from other repositories, like `jpackage`, `dag`, or `rpmforge`, you may encounter problems.
```

Comment:
No changes needed, I just didn't know that there was a warning admonition. Cool!